07a_system_activity_plots.R

```diff
@@ -189,75 +190,139 @@ renderSystemPlot <- function(id) {
         '#1b8297'
       )
          df <- system_activity_prep_detail()
+         mid_plot <- 4.5
        }
        
     s <- max(df$yend) + 20
     num_segments <- 20
     segment_size <- get_segment_size(s/num_segments)
-
+    
+    total_clients <- df %>%
+      filter(InflowOutflow == "Inflow") %>%
+      pull(values) %>%
+      sum()
+    
+    inflow_to_outflow <- df %>%
```

Comment:
NBD, but I would move this to below the validate, or even directly in the chart where you're printing "Total Change". That way, we don't have to run this code if we don't need to, i.e. if there's not enough data to show anyway.